src/peft/utils/save_and_load.py

```diff
@@ -219,6 +219,10 @@ def renamed_dora_weights(k):
     # ADDITIONAL TRAINING MODULES / MODULES_TO_SAVE
     for name, module in model.named_modules():
         if isinstance(module, AuxiliaryTrainingWrapper):
+            if name.startswith("_fsdp_wrapped_module."):
```

Comment:
I know the check is not technically required, but to me, it adds clarity to the intent. LMK if I should remove it though.